xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -122,10 +122,10 @@ class CActiveAESink : private CThread
   CEvent *m_inMsgEvent;
   int m_state;
   bool m_bStateMachineSelfTrigger;
-  std::chrono::milliseconds m_extTimeout;
-  std::chrono::milliseconds m_silenceTimeOut;
+  std::chrono::milliseconds m_extTimeout{std::chrono::milliseconds(0)};
```

Comment:
you could also do

```suggestion
  std::chrono::milliseconds m_extTimeout{std::chrono::milliseconds::zero()};
```

https://en.cppreference.com/w/cpp/chrono/duration/zero